bookkeeper-server/src/test/java/org/apache/bookkeeper/client/extension/TestContextExtension.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.bookkeeper.client.extension;
+
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * Unit Test Context Information.
+ */
+public class TestContextExtension implements BeforeEachCallback {
```

Comment:
I think we can put this common class into `testtools`, WDYT?

Reply:
> I think we can put this common class into `testtools`, WDYT?

Of course, I would also like this class to become more generalized, how do I add it to testtools please?

Reply:
This class isn't needed. Junit5 already contains [TestInfo](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/TestInfo.html) for this purpose. 
Please check https://junit.org/junit5/docs/current/user-guide/#writing-tests-dependency-injection for the example.